contracts/test/integration/IntegrationTestVoltMigrator.t.sol

```diff
@@ -315,4 +315,17 @@ contract IntegrationTestVoltMigratorTest is TimelockSimulation, vip13 {
             amountToTransfer
         );
     }
+
+    function testSweepNewVoltFails() public {
+        uint256 amountToSweep = voltV2.balanceOf(address(voltMigrator));
+
+        vm.startPrank(MainnetAddresses.GOVERNOR);
+        vm.expectRevert("VoltMigrator: cannot sweep new Volt");
+        voltMigrator.sweep(
+            address(voltV2),
+            MainnetAddresses.TIMELOCK_CONTROLLER,
+            amountToSweep
```

Comment:
To prove that the migration is locked for *any* destination and *amount*, you can add these values arguments and change the test to do fuzz/symbolic testing.

Reply:
Actually, I checked the foundry framework, and symbolic execution is not implemented right now, but you can still add the arguments to enable fuzz testing.